src/Compilers/CSharp/Portable/Symbols/Source/SourceClonedParameterSymbol.cs

```diff
@@ -164,6 +164,8 @@ internal override ImmutableHashSet<string> NotNullIfParameterNotNull
             get { return ImmutableHashSet<string>.Empty; }
         }
 
+        internal override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes => _originalParam.InterpolatedStringHandlerArgumentIndexes;
```

Comment:
>_originalParam.InterpolatedStringHandlerArgumentIndexes; [](http://example.com/codeflow?start=90&length=56)

Why is this needed? And why do we assume inxes won't change?

Reply:
Cloned parameter symbols share attribute data, so they shouldn't change.

Reply:
> Cloned parameter symbols share attribute data, so they shouldn't change.

Perhaps attribute applications shouldn't change, but the meaning  of attributes can. It is quite possible that for other well-known attributes it isn't an issue because they are self-contained. However for this attribute, surrounding parameters and parameter ordinals are significant, etc. So, I would say we should reinterpret the attribute application rather than take it as is. But the question is whether there is a scenario where this particular symbol is required to expose this information at all. If not, then why return anything but a value indicating - fail creating a handler at the call site. Anything other than an "error" value would require thorough testing for this code path.

Reply:
I'll make these throw for now and mark with a prototype, and we'll see if the indexes are needed during lowering in the next PR. I have a hunch they might be.